EdvClient.js

```diff
@@ -553,11 +553,10 @@ export class EdvClient {
    * @param {Object} options.invocationSigner - An API with an
    *   `id` property and a `sign` function for signing a capability invocation.
    *
-   * @return {Promise<Object>} resolves once the operation completes.
+   * @returns {Promise<undefined>} resolves once the operation completes.
```

Comment:
Defining the `Promise` will resolve to `undefined` seems odd.  And it looks odd here.  Seems it would be better to leave the behavior undefined, and if we later decide to return something, it wouldn't be a breaking change (for the odd person that depended on `undefined`.  It's difficult to determine what the jsdoc people want people to use in this regard.  Would it work to just use `{Promise}`?

Reply:
I vastly prefer the more explicit `{Promise<undefined}` as someone reading the docs knows the exact return type vs not being sure if the dev was just lazy and did not document the return type correctly. I have seen API docs which just stated {Promise} and then the promise resolved to something unexpected.